bundle/edu.gemini.wdba.xmlrpc.server/src/test/java/edu/gemini/wdba/tcc/Flamingos2SupportTest.java

```diff
@@ -71,14 +71,14 @@ public void testWavelength() throws Exception {
         // Use the disperser wavelength in spectroscopy mode with no filter
         flam2.setFilter(Filter.OPEN);
 
-        Pair[] dtA = new Pair[] {
-            new Pair<Disperser, String>(Disperser.R1200JH, "1.39"),
-            new Pair<Disperser, String>(Disperser.R1200HK, "1.871"),
-            new Pair<Disperser, String>(Disperser.R3000,   "1.65"),
+        final Pair[] dtA = new Pair[] {
```

Comment:
I don't recall but can you in Java declare an array with params like `Pair<Disperser, String>[]`